tabular/src/autogluon/tabular/models/catboost/callbacks.py

```diff
@@ -190,4 +190,4 @@ def after_iteration(self, info):
             self.best_score = cur_score
 
         should_stop = self.es.update(cur_round=info.iteration, is_best=is_best_iter)
-        return not should_stop
+        return not should_stop
```

Comment:
nit: It looks like your IDE may have removed a newline accidentally. We generally want to have newlines at the end of file to satisfy PEP.